packages/tools/client-debugger/client-debugger/src/DevtoolsLogger.ts

```diff
@@ -151,7 +154,10 @@ export class DevtoolsLogger extends TelemetryLogger {
 	 * @param event - The telemetry event to send.
 	 */
 	public send(event: ITelemetryBaseEvent): void {
-		// TODO: ability to disable the logger so this becomes a no-op
+		// Forward event to base logger
+		this.baseLogger?.send(event);
```

Comment:
Maybe wrap in `try-catch{ //Do nothing }`?

Reply:
Would that hide issues that we would otherwise hit if `send` failed on the inner logger? I can definitely see not wanting to have our devtools logic crash the application, but I would think we would want failures in their logger to do so.

Reply:
Hmm yeah good point. I was thinking mostly of not affecting the devtools logic, but for that I guess we put our calls first and ignore failures, then forward without any checks.